subsys/net/lib/lwm2m/lwm2m_rw_senml_json.c

```diff
@@ -89,8 +119,12 @@ struct json_out_formatter_data {
 	uint8_t writer_flags;
 	struct lwm2m_obj_path base_name;
 	bool add_base_name_to_start;
+	bool historical_data;
 	char bn_string[sizeof("/65535/65535/") + 1];
 	char name_string[sizeof("/65535/65535/") + 1];
+	char timestamp_buffer[42];
```

Comment:
Magic number

Reply:
Ok let me check do we have any definition for 64-bit time string